### ISSUE TITLE
Sync s3 to s3

### ISSUE THREAD

**Issue by nguyenvantruc** [peak/s5cmd#758]
Can I sync between two s3 storage? If so, what should the command look like?

**Comment by salim-b** [comment]
I don't think this is currently possible with `s5cmd` as we can only specify a single `--endpoint-url` / `S3_ENDPOINT_URL`. Only `s5cmd sync` between two different *regions* within the same object storage provider is currently possible.
